### PR TITLE
[ECS] Init

### PR DESCRIPTION
**Pull request by TomasVotruba**
Hey,

I've added GitHub Action, but it's not enabled here yet.
Could you enable it so we can see results?

### CODE REVIEW

File: ecs.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        __DIR__ . '/ecs.php',
+    ]);
+
+    $parameters->set(Option::SETS, [
+        SetList::SPACES,
+        // run and fix, one by one
+        // SetList::PSR_12,
+        // SetList::PHP_70,
+        // SetList::PHP_71,
+    ]);
```
Comment (TomasVotruba) [inline ecs.php:30]: The goal is to eventually add all the common sets, moreover PSR-12. 
But we should start with something simple and safe as "spaces" set.

The next sets will follow in next PRs after this get merged, to keep it simple.

**Comment by TomasVotruba** [comment]
Btw, have you considered switching to GitHub Actions? The Travis is super slow... still waiting on feedback :/ 

- Also has no features like GitHub Actions have.
- Also is [removing free minutes for open-source projects](https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works)

**Comment by BenMorel** [comment]
Thanks for the PR! I'm OK with removing the space between the closing parentheses and colon, however I'd like to preserve the aligned equal signs and `@param`s, is this possible? This is actually one of the few things that put me off when I tried ECS lately, I couldn't find a SetList that would work for me!

**Comment by TomasVotruba** [comment]
I haven't seen it such coding standard for a while, but I can try to ignore it.

**Comment by TomasVotruba** [comment]
Could you enable GitHub Actions first, so we have a feedback from CI if coding standard passes.

**Comment by BenMorel** [comment]
I'm new to GH actions, so I'd prefer if we can add it in a separate PR first!

**Comment by BenMorel** [comment]
I was reluctant at first regarding GH actions, because of the limits for open-source repos; but it sounds like Travis is becoming worse than that, so maybe now is the time to make the switch indeed.

**Comment by TomasVotruba** [comment]
I can't validate it without Github Actions.

> I was reluctant at first regarding GH actions, because of the limits for open-source repos; 

Huh, where is that information from? It's unlimited for open-source. We use it for a year on 10 projects with 20-30 pipelines each.

Travis is putting limit on open-source, closing down since November.

<br>

I can't validate if the code works or not without CI so I'm forced to close this PR for now.
Let me know when you've migrated to GitHub Actions.

**Comment by BenMorel** [comment]
This info comes from #45, but I haven't had a chance to verify it!

**Comment by BenMorel** [comment]
So it looks like you're right, I think the confusion came from the "GitHub Free" > 2,000 minutes per month. But as I understand it, this applies to private repos in the Free plan, not to public repos.

Do you want to help with migrating to GH actions by opening a PR to replicate the Travis config on GH?

**Comment by TomasVotruba** [comment]
I was about to only add ECS. Any other work is out of my free capacity.

**Comment by BenMorel** [comment]
I just moved the CI to GitHub Actions; I didn't have to enable anything though, do you think that the repo needed one workflow committed to start building workflows in PRs?

**Comment by TomasVotruba** [comment]
I'm sorry, I'm out of free time for this operation. It took more then I've expected.
But you can copy the PR here and merge it locally. The work is done.

**Comment by BenMorel** [comment]
> The work is done.

It's not, as far as I'm concerned: it's changing the current CS in a way that I think is detrimental to readability (un-aligning equal signs & `@param`s). But thanks anyway.

**Pull request closed without merging**